### ISSUE TITLE
Unused validation repeats value in v1-finetune.yaml

### ISSUE THREAD

**Issue by hopibel** [rinongal/textual_inversion#66]
The code for personalized.py only uses the repeats setting for the training set. Seems like the extra repeats setting under validation is a copy-paste oversight.

```python
# configs/stable-diffusion/v1-finetune.yaml

validation:
  target: ldm.data.personalized.PersonalizedBase
  params:
    size: 512
    set: val
    per_image_tokens: false
    repeats: 10  # <--- unused?
```

Speaking of which, what is the rationale for going over the dataset multiple times in an epoch? Does it improve training accuracy somehow or is it just a way to avoid running the validation test too often (10:1 ratio instead of 1:1)?

**Comment by CodeExplode** [comment]
On the topic of repeats, I wasn't sure if

data = instantiate_from_config(config.data)

Should be repeated twice in a row in https://github.com/rinongal/textual_inversion/blob/main/main.py#L749

**Comment by rinongal** [comment]
The repeats are used in both cases. They are a dirty hack to ensure that pytorch-lightning doesn't explode if you use a very small (1-2 image) dataset where your batch size is larger than the number of available images.

Since we're anyhow counting training steps and not epochs, and since the lengthy part of the validation step is independent of this value (the image outputs part), it shouldn't have much of an impact on your training times.

If you're using a very large training set you can lower the repeats value.

@CodeExplode That's a bug. I'll fix it.

**Comment by CodeExplode** [comment]
If we're trying 300+ training images, would a repeat of 1 seem reasonable?
